webapp/src/app/components/dialogs/map-settings/map-content-settings/map-content-settings.component.html

```diff
@@ -0,0 +1,67 @@
+<div class="content-container" fxLayout="column">
+	<!-- Map Name -->
+	<div fxLayout="row" fxFlex="0 0 auto">
+		<span fxFlexAlign="center">Map Name: </span>
+		<input type="text" class="default-input" [(ngModel)]="settings.name">
+	</div>
+	<hr>
+
+	<!--Map Size-->
+	<div fxLayout="row" fxFlex="0 0 auto">
+		<span fxFlexAlign="center" fxFlex="0 0 100px">Map Size:</span>
+		<span fxFlexAlign="center">width: </span>
+		<input type="number" class="default-input small-input" min="1" [ngModel]="settings.mapWidth" (change)="onNumberChange($event, 'mapWidth')">
+		<span fxFlexAlign="center">height: </span>
+		<input type="number" class="default-input small-input" min="1" [ngModel]="settings.mapHeight" (change)="onNumberChange($event, 'mapHeight')">
+	</div>
+	
+	<hr>
+	
+	<!--Levels-->
+	<div fxLayout="row" fxFlex="0 0 auto">
+		<span fxFlex="0 0 100px">Levels: </span>
+		<div fxLayout="column">
+			<div fxLayout="row" *ngFor="let level of settings.levels; let i=index">
+				<span fxFlexAlign="center">Level {{i}}:</span>
+				<input class="default-input" [(ngModel)]="level.height">
+				<button mat-button (click)="settings.levels.splice(i, 1)" tabindex="-1">Remove</button>
```

Comment:
Removing a layer does not properly clear the master level entry (even though it looks like it in the UI)